packages/blade/src/components/Icons/index.tsx

```diff
@@ -87,7 +87,7 @@ export { default as ExternalLinkIcon } from './ExternalLinkIcon';
 export { default as MailIcon } from './MailIcon';
 export { default as AlertCircleIcon } from './AlertCircleIcon';
 export { default as AlertOnlyIcon } from './AlertOnlyIcon';
-export { default as ScanIcon } from './ScanIcon';
+export { default as QRCodeIcon } from './QRCodeIcon';
```

Comment:
Would be a breaking change?

Reply:
Not a breaking change since ScanIcon was not present on any of the releases before. It was added only in https://github.com/razorpay/blade/pull/940/files#diff-ee401bacbc5df34a732325643f7e734ace96c88126b4f74ecc3b2200a81966daR1 (which isn't released)

Reply:
Plus this will be clubbed together with the v6 release anyways.